assets/src/stories-editor/helpers/index.js

```diff
@@ -1143,6 +1143,63 @@ export const maybeInitializeAnimations = () => {
 	}
 };
 
+/**
+ * Gets width and height delta values based on the original coordinates, rotation angle and mouse event.
+ *
+ * @param {Object} event MouseEvent.
+ * @param {number} angle Rotation angle.
+ * @param {number} lastSeenX Starting X coordinate.
+ * @param {number} lastSeenY Starint Y coordinate.
+ * @param {string} direction Direction of resizing.
+ * @return {Object} Width and height values.
+ */
+export const getDelta = ( event, angle, lastSeenX, lastSeenY, direction ) => {
+	let deltaH = 'bottom' === direction ? Math.sqrt( Math.pow( event.clientX - lastSeenX, 2 ) + Math.pow( event.clientY - lastSeenY, 2 ) ) : 0;
+	let deltaW = 'right' === direction ? Math.sqrt( Math.pow( event.clientX - lastSeenX, 2 ) + Math.pow( event.clientY - lastSeenY, 2 ) ) : 0;
```

Comment:
It might be useful to have small helper function for this, e.g. something like this:

```js
const getDelta = ( deltaX, deltaY ) => Math.sqrt( deltaX * deltaX + deltaY * deltaY );

let deltaH = 'bottom' === direction ? getDelta( event.clientX - lastSeenX, event.clientY - lastSeenY ) : 0;
let deltaW = 'right' === direction ? getDelta( event.clientX - lastSeenX, event.clientY - lastSeenY ) : 0;
```

It's not only DRY but also implicitly documented.